class/DatabaseConnection.php

```diff
@@ -0,0 +1,340 @@
+				VALUES
+				({$data_array['emp_id']}, 'residence', '{$data_array['r_street']}', 
+				'{$data_array['r_city']}', '{$data_array['r_state']}', '{$data_array['r_zip']}', 
+				'{$data_array['r_phone']}', '{$data_array['r_fax']}'),
+
+				({$data_array['emp_id']}, 'office', '{$data_array['o_street']}', 
+				'{$data_array['o_city']}', '{$data_array['o_state']}', '{$data_array['o_zip']}', 
+				'{$data_array['o_phone']}', '{$data_array['o_fax']}')";
+
+			$result_address = DatabaseConnection::db_query($q_address);
+
+			if(!$result_address)
+			{
+				log_error('Data insertion in address table');
+				echo "Data insertion into address table unsuccessful";
+				exit;
+			}
+
+			return self::$conn;
+		}		
+	}
+
+	/**
+	* To delete data from the database
+	*
+	* @access public
+	* @param  string $table_name, integer $id
+	* @return void
+	*/
+	public function delete_from_table($table_name, $id)
+	{
+		if($table_name == 'address')
+		{
+			$del_add = "DELETE FROM $table_name WHERE emp_id = " . $id;
+			DatabaseConnection::db_query($del_add);
+		}
+		elseif($table_name == 'employee')
+		{
+			$del_emp = "DELETE FROM $table_name WHERE id = " . $id;
+			DatabaseConnection::db_query($del_emp);
+		}
+	}
+
+	/**
+	* To delete the employee profile picture
+	*
+	* @access public
+	* @param  integer $id
+	* @return void
+	*/
+	public function delete_pic($id)
+	{
+		$sel_pic = "SELECT photo FROM employee WHERE id = " . $id;
+		$pic_object = DatabaseConnection::db_query($sel_pic);
+		$row = DatabaseConnection::db_fetch_array($pic_object);
+		$pic_location = PIC_PATH . $row['photo'];
+		unlink($pic_location);
+	}
+
+	/**
+	* To display the employee list
+	*
+	* @access public
+	* @param  void
+	* @return object
+	*/
+	public function display_list()
+	{
+		$q_list = "SELECT emp.prefix AS prefix, 
+			CONCAT(emp.first_name,' ', emp.middle_name,' ', emp.last_name) AS name, 
+			emp.gender AS gender, emp.dob AS dob, 
+			emp.marital_status AS marital_status, emp.employment AS employment, 
+			emp.employer AS employer, 
+			CONCAT(res.street, ', ', res.city, ', ', res.state, ', ', res.zip, ', ', 
+			res.phone, ', ', res.fax) AS res_add, 
+			CONCAT(off.street, ', ', off.city, ', ', off.state, ', ', off.zip, ', ', 
+			off.phone, ', ', off.fax) AS off_add, emp.comm_id AS comm_id, emp.id AS id, 
+			emp.photo AS photo from employee AS emp 
+			INNER JOIN address AS res ON (emp.id = res.emp_id AND res.address_type = 'residence')
+			INNER JOIN address AS off ON (emp.id = off.emp_id AND off.address_type = 'office')";
+
+		$result_list = DatabaseConnection::db_query($q_list);
+		return $result_list;
+	}
+
+	public function select_comm_field($value)
+	{
+		$q_comm = "SELECT GROUP_CONCAT(type SEPARATOR ', ') FROM `communication_medium` 
+		WHERE id IN ($value)";
+
+		$result = DatabaseConnection::db_query($q_comm);
+		return $result;
+	}
+
+	/**
+	* To display the employee list
+	*
+	* @access public
+	* @param  string $table_name, array $data_array
```

Comment:
Function has two parameters, specify both in function header
@param string $email
@param integer $id